src/app/exams/exams-view.component.ts

```diff
@@ -61,12 +68,12 @@ export class ExamsViewComponent implements OnInit, OnDestroy {
       this.previewMode = params.get('preview') === 'true' || this.isDialog;
       this.questionNum = +params.get('questionNum') || this.questionNum;
       if (this.previewMode) {
-        ((this.exam || this.submission) ? of({}) : this.couchService.get(`exams/${params.get('examId')}`)).subscribe((res) => {
+  ((this.exam || this.submission) ? of({}) : this.couchService.get(`exams/${params.get('examId')}`)).subscribe((res) => {
```

Comment:
Please be more careful when making changes.  This line and line 76 should not have the indentation changed.